lib/presentation/mixins/connect_page_mixin.dart

```diff
@@ -295,6 +295,15 @@ mixin ConnectPageMixin {
   }) {
     final loginTokenExisted = getQueryParameter('loginToken') != null;
     if (!loginTokenExisted) return;
-    html.window.history.replaceState({}, '', '/#/${route ?? 'rooms'}');
+    html.window.history.replaceState(
+      {},
+      '',
+      '${getBaseUrlBeforeHash(html.window.location.href)}#/${route ?? 'rooms'}',
+    );
+  }
+
+  String getBaseUrlBeforeHash(String fullUrl) {
+    final fragmentIndex = fullUrl.indexOf('#/');
+    return fragmentIndex != -1 ? fullUrl.substring(0, fragmentIndex) : fullUrl;
```

Comment:
we should separate this fix in other PR and test carefully with some platform (saas and normal)

Reply:
Move to this ticket #2067